package.json

```diff
@@ -59,7 +60,7 @@
     "expo-status-bar": "~1.11.1",
     "expo-system-ui": "^2.9.4",
     "expo-task-manager": "~11.7.3",
-    "expo-updates": "^0.25.9",
+    "expo-updates": "~0.24.12",
```

Comment:
For reviewer: This was due to incorrect version being pushed into main. For current Expo version, 0.24.12 is desired one.

Reply:
this has been resolved on main branch already, would recommend rebasing/updating this PR